docker-images/artifacts/kafka-thirdparty-libs/3.2.x/pom.xml

```diff
@@ -130,14 +130,16 @@
         </dependency>
         <dependency>
             <groupId>io.opentelemetry</groupId>
-            <artifactId>opentelemetry-exporter-jaeger</artifactId>
+            <artifactId>opentelemetry-exporter-otlp</artifactId>
             <version>${opentelemetry.version}</version>
+            <!--
             <exclusions>
                 <exclusion>
                     <groupId>com.fasterxml.jackson.core</groupId>
                     <artifactId>jackson-core</artifactId>
                 </exclusion>
             </exclusions>
+            -->
```

Comment:
This should be deleted or user?

Reply:
Yeah, I commented just to check that the build was ok this time but then forgot to remove.